### PR TITLE
Reviewed Readme.md and step by step guide. Issue #67

### PR DESCRIPTION
**Pull request by beatrizRG**
Found different issues during step by step guide execution.
1. Readme.md updated with the new steps ( remove @types/core-js and added some missing information)
2. Updated config files in order to be consistent to the changes done in the readme.md file.
3. Added a new image displaying a new warning found and how to avoid it.

### CODE REVIEW

File: 02 Fx/03 Angular 2/README.md [hunk 1]
```diff
@@ -63,14 +59,36 @@ npm install @types/core-js --save-dev
...
+```diff
+{
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "declaration": false,
+    "noImplicitAny": false,
+    "sourceMap": true,
+-   "noLib": false,
+   "suppressImplicitAnyIndexErrors": true
```
Comment (nasdan) [inline 02 Fx/03 Angular 2/README.md]: Add comma to the end of line:

you can add like:

```
- "suppressImplicitAnyIndexErrors": true
+ "suppressImplicitAnyIndexErrors": true,
+ "lib": ["dom", "es2015"]
```

File: 02 Fx/03 Angular 2/README.md [hunk 2]
```diff
@@ -191,13 +209,10 @@ module.exports = {
     "declaration": false,
     "noImplicitAny": false,
     "sourceMap": true,
-    "noLib": false,
 -   "suppressImplicitAnyIndexErrors": true
```
Comment (nasdan) [inline 02 Fx/03 Angular 2/README.md]: I think this line should be looks like:

```
  "suppressImplicitAnyIndexErrors": true,
- "lib": ["dom", "es2015"]
+ "lib": ["dom", "es2015"],
+ "experimentalDecorators":true

File: 02 Fx/03 Angular 2/tsconfig.json [hunk 3]
```diff
@@ -5,11 +5,10 @@
     "declaration": false,
     "noImplicitAny": false,
     "sourceMap": true,
-    "noLib": false,
     "suppressImplicitAnyIndexErrors": true,
     "experimentalDecorators": true,
+    "lib": ["dom", "es2015"]   ,
```
Comment (nasdan) [inline 02 Fx/03 Angular 2/tsconfig.json]: Remove extra space before comma

File: 02 Fx/03 Angular 2/README.md [hunk 4]
```diff
@@ -306,7 +352,7 @@ npm install @types/webpack-env --save-dev
```
Comment (Nasdan) [inline 02 Fx/03 Angular 2/README.md]: In this [section](https://github.com/Lemoncode/webpack-2.x-by-sample/pull/84/files#diff-bd772c9c75aaa0910796cbd1bdf95057R341), it should looks like:

```
  "lib": ["dom", "es2015"],
- "experimentalDecorators":true
+ "experimentalDecorators":true,
+ "types": [
+   "webpack-env"
+ ]
```

**Comment by nasdan** [comment]
Great stuff! Only some comments about tsconfig

File: 02 Fx/03 Angular 2/README.md [hunk 5]
```diff
@@ -5,19 +5,19 @@ In this sample we are going to setup a basic Angular 2 app with Webpack.
...
 
 Summary steps:
- - Installing Angular 1.x libraries.
- - Creating the app.
- - Instantiating the app from the html.
- - Creating a component (inline HTML).
- - Creating a service.
- - Displaying the component.
- - Creating an external HTML template and consuming it.
+ - Install Angular 1.x libraries.
```
Comment (nasdan) [inline 02 Fx/03 Angular 2/README.md:8]: If we change `Summary steps` I think we have to change [root Readme](https://github.com/Lemoncode/webpack-2.x-by-sample/blob/master/README.md)

File: 02 Fx/03 Angular 2/README.md (hunk 4 above)
Comment (nasdan) [inline 02 Fx/03 Angular 2/README.md:352]: In this [section](https://github.com/Lemoncode/webpack-2.x-by-sample/pull/84/files#diff-bd772c9c75aaa0910796cbd1bdf95057R341), it should looks like:

```
  "lib": ["dom", "es2015"],
- "experimentalDecorators":true
+ "experimentalDecorators":true,
+ "types": [
+   "webpack-env"
+ ]
```

**Comment by beatrizRG** [comment]
I have included the changes that you have commented.
Root readme changes. Update local readme.
Update tsconfig file.

**Comment by nasdan** [comment]
I think its ready to merge, isn't it? Did you close PR by mistake?

**Review by nasdan — APPROVED** [review]

**Pull request merged**